userProfile/userProfile.mjs

```diff
@@ -0,0 +1,67 @@
+
+import DatabaseDriver from "../database/driver.mjs"
+
+// This module will use the TinyPEN API (RERUM Mongo DB)
+const database = new DatabaseDriver("tiny")
```

Comment:
tiny is not for the internal profile information

Reply:
changed , updated.